CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ option(TARGET_NATIVE_CLIENT "Include Native Client" OFF)
 option(TARGET_X86 "Include x86 target" ON)
 option(TARGET_ARM "Include ARM target" ON)
 option(TARGET_AARCH64 "Include AARCH64 (arm64) target" ON)
+option(TARGET_MIPS "Include MIPS (\"Mips\" in LLVM) target" ON)
```

Comment:
The Mips in LLVM comment seems unnecessary and a little confusing. People compiling Halide with cmake may not also be compiling llvm.